python/ray/tune/log_sync.py

```diff
@@ -103,11 +111,13 @@ def sync_now(self, force=False):
             if not distutils.spawn.find_executable("rsync"):
                 print("Error: log sync requires rsync to be installed.")
                 return
+            source = '{}@{}:{}/'.format(ssh_user, self.worker_ip,
+                                        escape_rsync_location(self.local_dir))
+            target = '{}/'.format(escape_rsync_location(self.local_dir))
```

Comment:
I don't know if this is ideal or needed. I tested several ways of quoting the local dir without escaping and didn't manage to make it work. Happy to change this if someone has better ideas of how to handle it.

Reply:
This is actually weird since the log sync definitely worked before with whitespaces and some other special characters. Not sure why the parenthesis caused a problem.

Reply:
Do you have some example input / outputs of this with other quoting strategies?